scientific_library/tvb/datatypes/projections.py

```diff
@@ -32,14 +32,17 @@
 
 .. moduleauthor:: Lia Domide <lia.domide@codemart.ro>
 """
+from enum import Enum
 
 from tvb.basic.readers import try_get_absolute_path, FileReader
 from tvb.datatypes import surfaces, sensors
 from tvb.basic.neotraits.api import HasTraits, Attr, NArray
 
-EEG_POLYMORPHIC_IDENTITY = "projEEG"
-MEG_POLYMORPHIC_IDENTITY = "projMEG"
-SEEG_POLYMORPHIC_IDENTITY = "projSEEG"
+
+class ProjectionsType(Enum):
+    EEG_POLYMORPHIC_IDENTITY = "projEEG"
+    MEG_POLYMORPHIC_IDENTITY = "projMEG"
+    SEEG_POLYMORPHIC_IDENTITY = "projSEEG"
```

Comment:
Maybe remove the suffix `_POLYMORPHIC_IDENTITY` from inside the ENUM item names ?